.template/addons/terraform/heroku/heroku.yml

```diff
@@ -0,0 +1,5 @@
+build:
+  docker:
+    web: Dockerfile
+run:
+  web: bin/start.sh
```

Comment:
How about the Worker? Do we need to start the worker? @namtx 🤔

Reply:
good point, `worker` is already added.